getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,249 @@
+---
+layout: default
+title: Installing Community
+published: true
+sorting: 80
+---
+
+These instructions describe how to download and install the latest version of CFEngine Community in a docker containerized environment using pre-compiled rpm packages for ubi9 images.
```

Comment:
I don't know of any other place we reference ubi images. Might be worth some description of what a ubi9 image is.

> Red Hat Universal Base Images (UBI) are OCI-compliant container base operating system images with complementary runtime languages and packages that are freely redistributable.

Reply:
Added these references:
- [RedHat Universal Base Image (UBI)](https://www.redhat.com/en/blog/introducing-red-hat-universal-base-image)
- [Using the UBI init images](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/building_running_and_managing_containers/index#using-the-ubi-init-images_assembly_adding-software-to-a-ubi-container)
- [ubi9-init repository](https://catalog.redhat.com/software/containers/ubi9-init/6183297540a2d8e95c82e8bd)